static/app/utils/queryClient.tsx

```diff
@@ -241,35 +217,6 @@ export function setApiQueryData<TResponseData>(
   return newResponse[0];
 }
 
-/**
- * This method, given an `api` will return a new method which can be used as a
- * default `queryFn` with `useInfiniteQuery` hook.
- *
- * This returned method, the `queryFn`, unwraps react-query's `QueryFunctionContext`
- * type into parts that will be passed into api.requestPromise including the next
- * page cursor.
- *
- * See also: fetchDataQuery & fetchMutation
- */
-export function fetchInfiniteQuery<TResponseData>(api: Client) {
```

Comment:
this wasn’t used outside this file so I just inlined it into `useInfiniteApiQuery`.